GSoC-Participants.md

```diff
@@ -0,0 +1,135 @@
+---
+layout: default
+title: GSoC participants
+---
+
+This document collects the list of contributors who've contributed
+to Git via GSoC.
+
+<!-- [ [project]() ] [ [final report]() ] [ [blog]() ] -->
+
+### 2023
+
+*TODO*
+
+#### References
+
+- [GSoC archive](https://summerofcode.withgoogle.com/archive/2023/organizations/git)
+
+### 2022
+
+*TODO*
+
+#### References
+
+- [GSoC archive](https://summerofcode.withgoogle.com/archive/2022/organizations/git)
```

Comment:
From _posts/2022-06-30-edition-88.markdown:

  - Abhradeep Chakraborty, who works on
    ["Reachability bitmap improvements"](https://summerofcode.withgoogle.com/programs/2022/projects/UPtA6qdf),
    has published [his latest post](https://medium.com/@abhra303/gsoc-week-2-redesign-the-table-format-829dae755a5)
    on [his Medium blog](https://medium.com/@abhra303).

  - Shaoxuan Yuan, who works on ["More Sparse Index Integrations"](https://summerofcode.withgoogle.com/programs/2022/projects/hz4rcOUB)
    has published [his latest post](https://ffyuanda.github.io/blog/GSoC-week-2/)
    on [his website](https://ffyuanda.github.io/).

  - Jaydeep Das, who works on ["Unify ref-filters with other pretty formats"](https://summerofcode.withgoogle.com/programs/2022/projects/8mYcnerl)
    has published [his latest post](https://jdeep.me/posts/10.d/)
    on [his website](https://jdeep.me/posts/).